Sources/SwiftFormatPrettyPrint/TokenStreamCreator.swift

```diff
@@ -1240,9 +1248,10 @@ fileprivate final class TokenStreamCreator: SyntaxVisitor {
 
   override func visit(_ node: TupleTypeElementSyntax) -> SyntaxVisitorContinueKind {
     before(node.firstToken, tokens: .open)
-    after(node.colon, tokens: .break)
-    after(node.inOut, tokens: .break)
```

Comment:
Do we need this break after the `inOut` token?

Reply:
The `inOut` token here is legacy, I believe, from when the attribute used to come before the variable name instead of the type (e.g., `inout foo: Bar` instead of `foo: inout Bar`). The parser doesn't accept that syntax anymore (I tried it in the syntax explorer), so I don't think this will ever be non-nil. When using the new syntax (`foo: inout Bar`), the type comes across as an `AttributedTypeSyntax`, and we add a break after the `specifier` (which might be `inout`, `__shared`, etc.).